service/src/main/java/uk/nhs/adaptors/gp2gp/configuration/MongoClientConfiguration.java

```diff
@@ -63,10 +63,10 @@ private String createConnectionString() {
             LOGGER.info("A value was provided from mongodb host. Generating a connection string from individual properties.");
             return createConnectionStringFromProperties();
         } else if (StringUtils.isNotBlank(uri)) {
-            LOGGER.info("A mongodb connection string provided in spring.data.mongodb.uri and will be used to configure the database connection.");
+            LOGGER.info("A mongodb connection string provided in spring.data.mongodb.uri will be used for database connection.");
             return uri;
         } else {
-            LOGGER.error("Mongodb must be configured using a connection string or individual properties. Both uri and host are null or empty");
+            LOGGER.error("Both spring.data.mongodb.uri and spring.data.mongodb.host are null or empty.");
```

Comment:
`spring.data.mongodb.uri` won't make sense to the GP Supplier using the adaptor. If we're going to use property names instead of a description they should be the environment variables GP2GP_MONGO_URI and GP2GP_MONGO_HOST